app/build.gradle

```diff
@@ -5,7 +5,7 @@ android {
     buildToolsVersion "29.0.2"
     defaultConfig {
         applicationId "com.adit.tictactoe"
-        minSdkVersion 15
```

Comment:
Why have we changed the midSdkVersion ?

Reply:
The reason for change for minSdkversion is that creation of new activity requires a minimum SDK version in Android Studio or so i was notified. So,, had to change it.

Reply:
Fine, we can keep it as 21 then.